ndio/remote/neurodata.py

```diff
@@ -22,9 +22,9 @@
 except ImportError:
     import urllib2
 
-DEFAULT_HOSTNAME = "openconnecto.me"
+DEFAULT_HOSTNAME = "cloud.neurodata.io"
 DEFAULT_SUFFIX = "nd"
-DEFAULT_PROTOCOL = "https"
+DEFAULT_PROTOCOL = "http"   #should be https
```

Comment:
This should remain http for now. Essentially the issue is that public tokens should still be accessible from http. We can talk more about this offline.